backend/app/modules/fact_check_tool.py

```
@@ -0,0 +1,166 @@
+    
+    if not queries:
+        return {"search_results": []}
+
+    async def run_one_search(q):
+        try:
+            query_str = q.get("query")
+            c_id = q.get("claim_id")
+            
+            res = await asyncio.to_thread(search_tool.invoke, query_str)
+            logger.info(f"Search Result for Claim {c_id}: {res[:200]}...")
+            return {"claim_id": c_id, "result": res}
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)